registry.k8s.io/images/charts/images.yaml

```diff
@@ -1 +1,5 @@
 # TODO: add images
```

Comment:
I would delete this TODO at this point :)

Reply:
Good point 😉

Reply:
Removed.